dipy/workflows/stats.py

```diff
@@ -543,6 +548,11 @@ def run(self, subject_folder, threshold=6, out_dir=''):
 
         """
         rng = np.random.RandomState()
+
+        clust_thr = list(map(float, clust_thr.strip('[]').split(',')))
```

Comment:
you do not need that if you use `variable float`

Reply:
but it is a list
if I just write float
it gives error

Reply:
let me try one more time

Reply:
`variable float` not just `float` and you will get your list.